src/bip44.c

```diff
@@ -72,8 +72,20 @@ bool bip44_hasShelleyPrefix(const bip44_path_t* pathSpec)
 #undef CHECK
 }
 
+bool bip44_hasMultisigPrefix(const bip44_path_t* pathSpec)
+{
+#define CHECK(cond) if (!(cond)) return false
+	CHECK(pathSpec->length > BIP44_I_COIN_TYPE);
+	CHECK(pathSpec->path[BIP44_I_PURPOSE] == (PURPOSE_MULTISIG | HARDENED_BIP32));
+	CHECK(pathSpec->path[BIP44_I_COIN_TYPE] == (ADA_COIN_TYPE | HARDENED_BIP32));
+	return true;
+#undef CHECK
+}
+
 bool bip44_hasValidCardanoWalletPrefix(const bip44_path_t* pathSpec)
 {
+	// TODO should bip44_hasMultisigPrefix come here? does it fulfill the
```

Comment:
This function is used very rarely. I would consider removing it altogether, but not in case it would lead to annoyingly long if/assertion conditions.  I'd remove "Valid" from the name and include multisig here (at least it seems to me it is used that way).